Drivers/SHT30/sht30.h

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file sht30.h
+ * @brief 
+ */
+#ifndef SHT30_H
+#define SHT30_H
+
+#include <stdint.h>
+
+//SHT30-DIS Humidity Sensor
+/**
+ * https://www.mouser.com/datasheet/2/682/Sensirion_Humidity_Sensors_SHT3x_Datasheet_digital-971521.pdf  --Datasheet
+ * 
+ */
+
+#define SWITCHBYTES(x)             (((x<<8) | (x>>8)) & 0xFFFF)      //Switches the first and second byte (This is required for all command values)
+
+#define SHT30_I2C_ADDR             0x44     //If ADDR (pin2) is connected to VDD, 0x45
+
+#define SHT30_START_CMD_WCS        0x2C06   //Start measurement command with clock streching enabled and high repeatability
+#define SHT30_START_CMD_NCS        0x2400   //Start measurement command with clock streching disabled and high repeatability 
+#define SHT30_READSTATUS           0xF32D   //Read Out of Status Register
+#define SHT30_CLEARSTATUS          0x3041   //Clear Status
+#define SHT30_SOFTRESET            0x30A2   //Soft Reset
+#define SHT30_HEATEREN             0x306D   //Heater Enable
+#define SHT30_HEATERDIS            0x3066   //Heater Disable
+
+#define SHT30_REG_HEATER_BIT       0x0d     //Status Register Heater Bit
+
+typedef struct
+{
+    uint16_t cmdVal;
+    uint8_t cmd2Bytes[2];
+} cmd_t;
+
+typedef struct
+{
+    uint16_t statusReg;
```

Comment:
We should read thru the datasheet and see what else we might want to add in this. We also need to investigate the STM32 I2C HAL and see if we need references to any internal I2C bus addresses or struct-based APIs like regmap in the Linux kernel